lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -363,6 +392,35 @@ module Resources
       'relationship': [
         {
           'id': 'CI-97982588-0-0',
+          'extension': [
+            {
+              'valuePeriod': {
+                'start': '2016-12-19T16:44:25.000Z',
+                'end': '2022-05-01T16:56:56.000Z'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            },
+            {
+              'valueCodeableconcept': {
+                'coding': [
```

Comment:
Having sister as the relation here doesn't make sense. I think it should be mother.

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/624/commits/f8e3e624c57363c311b7e4577162e926522db9c8#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR408-R409 
https://github.com/cerner/fhir.cerner.com/pull/624/commits/f8e3e624c57363c311b7e4577162e926522db9c8#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR414-R415

Reply:
changed here f8e3e62#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR408-R409
f8e3e62#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR414-R415